build/script.js

```diff
@@ -102,6 +105,34 @@ gulp.task('scripts-watch', createScriptsStream(false));
  * 'async'.
  */
 function createCompileTask(translation) {
+
+  let outputDir =
+      translation ? path.join(conf.paths.i18nProd, `/${translation.key}`) : conf.paths.prodTmp;
+
+  return (
+      (next) =>
+          gulp.src([
+                // Application source files.
+                path.join(conf.paths.frontendSrc, '**/*.js'),
+                // Partials generated by other tasks, e.g., Angular templates.
+                path.join(conf.paths.partials, '**/*.js'),
+                // Include base.js to enable some compiler functions, e.g., @export annotation
+                // handling and getMsg() translations.
+                path.join(
+                    conf.paths.bowerComponents, 'google-closure-library/closure/goog/base.js'),
+              ])
+              .pipe(gulpIf('src/app/frontend/common/appconfig/appconfig_service.js',gulpReplace('BUILD_DASHBOARD_VERSION', conf.deploy.version.release)))
+              .pipe(gulpIf('src/app/frontend/common/appconfig/appconfig_service.js',gulpReplace('BUILD_GIT_COMMIT', process.env.TRAVIS_COMMIT)))
+              .pipe(closureCompiler(translation))
```

Comment:
I have copied the path from gulp-print, but it is not working. 

I guess the problem is that closure compiler does not take the result from gulp replace, but loads the files from disk and discards my changes. 

Maybe because it is not a real gulp task but just a wrapper around a java program?